src/test/java/wolox/training/entitiesTest/EntitiesTest.java

```diff
@@ -0,0 +1,74 @@
+package wolox.training.entitiesTest;
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.